charts/datadog/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Datadog changelog
 
+## 2.31.0
+
+***Warning:*** From this version onwards, on GKE Autopilot, only one "datadog" Helm chart release is allowed by the Kubernetes namespace due to the following new constraints:
```

Comment:
Would it be possible to have the hardcoded-named service account be created by a first Helm release and re-used by a second Helm release that would be created with:
```yaml
agents:
  rbac:
    create: false
    serviceAccountName: datadog-agent
```

Technically speaking, is it more correct to say that only one Helm release on GKE auto-pilot can leave `agents.rbac.create` set to its default `true` value ?
I ask this, in the hypothesis where we wouldn’t enforce this restriction with the `fail` statement in the `NOTES.txt`.

Reply:
Also
```suggestion
***Warning:*** From this version onwards, on GKE Autopilot, only one "datadog" Helm chart release is allowed by Kubernetes namespace due to the following new constraints:
```

Reply:
I will keep the restriction as it is because, the serviceAccount in one think but we also have the issue with the install-info configmap that need to be unique.
So I think it is better to say for now `one release per namespace`